app/mailers/user_mailer.rb

```diff
@@ -64,4 +65,19 @@ def sign_in_email(user:, url:, token_expiry:)
       },
     )
   end
+
+  def mentor_sign_in_reminder_email(user)
+    sign_in_url = Rails.application.routes.url_helpers.new_user_session_url(host: Rails.application.config.domain,
+                                                                            **UtmService.email(:new_mentor))
```

Comment:
> **UtmService.email(:new_mentor))

Is this true? I think not. Let's make a new campaign for it, like `:mentor_sign_in_reminder`?

Reply:
good spot! completely missed this.